test/PreAMMBatcher.e2e.ts

```diff
@@ -160,27 +161,29 @@ describe("PreAMMBatcher-e2e", () => {
         [walletTrader1, walletTrader2],
         [walletTrader3, walletTrader4],
       ),
-      true,
+      false,
     );
```

Comment:
Since this is a debug flag for a testing function, it could be removed and the `debug` package could be used internally by this function.

Reply:
Done!